datasets/reddit/README.md

```diff
@@ -0,0 +1,80 @@
+# Reddit
+
+## Dataset Description
+
+The Reddit dataset is a graph dataset from Reddit posts made in the month of September, 2014. The node label in this case is the community that a post belongs to. 50 large communities have been sampled to build a post-to-post graph, connecting posts if the same user comments on both. 
+
+Statistics:
+- Nodes: 232965
+- Edges: 114615892
+- Number of Classes: 41
+
+#### Citation
+- Original Source
+  + [Website](http://snap.stanford.edu/graphsage/)
+  + LICENSE: [MIT](https://github.com/williamleif/GraphSAGE/blob/master/LICENSE.txt)
+```
+@article{hamilton2017inductive,
+  title={Inductive representation learning on large graphs},
+  author={Hamilton, Will and Ying, Zhitao and Leskovec, Jure},
+  journal={Advances in neural information processing systems},
+  volume={30},
+  year={2017}
+}
+```
+
+- Current Version
+  + [Website](http://snap.stanford.edu/graphsage/)
+  + LICENSE: [MIT](https://github.com/williamleif/GraphSAGE/blob/master/LICENSE.txt)
+```
+@article{hamilton2017inductive,
+  title={Inductive representation learning on large graphs},
+  author={Hamilton, Will and Ying, Zhitao and Leskovec, Jure},
+  journal={Advances in neural information processing systems},
+  volume={30},
+  year={2017}
+}
+```
+
+- Previous Version
+  + [Website](https://drive.google.com/drive/folders/1rq-H0XUM0BIRW9Pq5P4FMC9Xirpdx6zs)
+  + LICENSE: [MIT](https://github.com/GraphSAINT/GraphSAINT/blob/master/LICENSE)
+```
+@article{zeng2019graphsaint,
+  title={Graphsaint: Graph sampling based inductive learning method},
+  author={Zeng, Hanqing and Zhou, Hongkuan and Srivastava, Ajitesh and Kannan, Rajgopal and Prasanna, Viktor},
+  journal={arXiv preprint arXiv:1907.04931},
+  year={2019}
+}
```

Comment:
Not sure if it is okay to have previous version newer than current version. @jiaqima What do you think?

Reply:
For the previous version, it's more like a variant (sparse version) for original Reddit Dataset. But the original version seems more widely used still. I have the same confusion with Xingjian and I'm open to get advice.

Reply:
Where exactly does this version of dataset come from? If it comes from the 2017 version and has nothing to do with the 2019 paper, we should remove the 2019 paper. It's ok to leave previous versions blank.

Reply:
It comes from 2017 version. Let me remove 2019 paper for the previous version.